onnx/common/ir_pb_converter.cc

```diff
@@ -20,6 +20,7 @@ Tensor tensorProtoToTensor(const ONNX_NAMESPACE::TensorProto & tp) {
   switch(tp.data_type()) {
   case ONNX_NAMESPACE::TensorProto_DataType_FLOAT:
   case ONNX_NAMESPACE::TensorProto_DataType_COMPLEX64: {
+    // TODO: fix this handling
```

Comment:
what is the issue here? should we fix it now, or maybe open a task to fix it later?

Reply:
I think this is fine. Never mind. :-)